### PR TITLE
Adding TFIDF

### PR DESCRIPTION
**Pull request by NavyaTatikonda**

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ See [Notebooks from NVIDIA's RAPIDS](../cloud-devops-infra/gpus/rapids.md#notebo
     - [Spacy/Texacy Examples](../examples/better-nlp/notebooks/google-colab/better_nlp_spacy_texacy_examples.ipynb)
     - [Summarisers Examples](../examples/better-nlp/notebooks/google-colab/better_nlp_summarisers.ipynb)
 - [Bag of words notebook](../blogs/keras-bag-of-words-expanded-version/data-scripts-notebooks/)
+- [TFIDF notebook](../I shall update here.)
```
Comment (neomatrix369) [inline notebooks/README.md:56]: @NavyaTatikonda Yes pls proceed - sensible place, do you want to close the other PR?

Comment (NavyaTatikonda) [reply notebooks/README.md:56]: Yes, doing it now :)

Comment (NavyaTatikonda) [reply notebooks/README.md:56]: Will you merge this PR?

Comment (neomatrix369) [reply notebooks/README.md:56]: Let me know when it\s ready with the link

Comment (neomatrix369) [reply notebooks/README.md:56]: @NavyaTatikonda Any update on this?

**Comment by NavyaTatikonda** [comment]
I have the doc ready in Scikit-learn, but not in keras. 
Can I still proceed and update it?

File: notebooks/README.md (hunk 1 above)
Comment (NavyaTatikonda) [reply notebooks/README.md:56]: I have the doc ready in Scikit-learn, but not in keras.
Can I still proceed and update it?

Comment (neomatrix369) [reply notebooks/README.md:56]: ok

Comment (neomatrix369) [reply notebooks/README.md:56]: Submit whatever is ready

**Comment by NavyaTatikonda** [comment]
can you check if this can be added - https://github.com/NavyaTatikonda/awesome-ai-ml-dl/blob/master/blogs/TFIDF%20Scikit-learn.ipynb

**Pull request merged**